content/golems/getting-started.md

```
@@ -0,0 +1,179 @@
+bun src/email-golem/index.ts
+
+# Start the Telegram bot
+bun src/telegram-bot.ts
+
+# Trigger night shift improvements
+bun src/night-shift.ts
+```
+
+## Monorepo Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)